### PR TITLE
Retain arity in memoize and once

### LINKED ISSUE #1519: memoize and once don't retain arity
Functions returned by memoize always hava 0-arity, leading to unexpected results when used as the first step in a pipeline.

### PR DESCRIPTION
**Pull request by asaf-romano**
Closes #1519

### CODE REVIEW

**Comment by CrossEye** [comment]
:herb:

File: src/once.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
+var _arity = require('./internal/_arity');
 var _curry1 = require('./internal/_curry1');
 
-
```
Comment (davidchambers) [inline src/once.js]: Can we keep this line, please? :)

File: test/memoize.js [hunk 2]
```diff
@@ -85,4 +85,8 @@ describe('memoize', function() {
     eq(count, 1);
   });
 
+  it('retains arity', function() {
+    var f = R.memoize(function(a, b) { return a + b; });
+    eq(f.length, 2);
+  });
 });
```
Comment (davidchambers) [inline test/memoize.js]: Please add an empty line between lines 91 and 92. :)

File: test/once.js [hunk 3]
```diff
@@ -31,4 +31,8 @@ describe('once', function() {
     eq(ctr, 1);
   });
 
+  it('retains arity', function() {
+    var f = R.once(function(a, b) { return a + b; });
+    eq(f.length, 2);
+  });
 });
```
Comment (davidchambers) [inline test/once.js]: Ditto.

**Comment by davidchambers** [comment]
Looks great, aside from the whitespace changes! ;)

**Comment by davidchambers** [comment]
:deciduous_tree: Very nice!

**Pull request merged**